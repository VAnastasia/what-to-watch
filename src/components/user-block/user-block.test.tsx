import * as React from "react";
import * as renderer from "react-test-renderer";
import {MemoryRouter} from "react-router-dom";
import UserBlock from "./user-block";

it(`UserBlock component render correctly`, () => {
  const tree = renderer.create(
      <MemoryRouter>
        <UserBlock
          authorizationStatus={`AUTH`}
          avatarUrl={`avatar`}
        />
      </MemoryRouter>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
