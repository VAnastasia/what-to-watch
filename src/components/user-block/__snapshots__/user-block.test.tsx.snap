// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserBlock component render correctly 1`] = `
<div
  className="user-block"
>
  <a
    href="/mylist"
    onClick={[Function]}
  >
    <div
      className="user-block__avatar"
    >
      <img
        alt="User avatar"
        height="63"
        src="avatar"
        width="63"
      />
    </div>
  </a>
</div>
`;
